src/wp-content/themes/twentyfourteen/header.php

```diff
@@ -9,24 +9,13 @@
  * @since Twenty Fourteen 1.0
  */
 ?><!DOCTYPE html>
-<!--[if IE 7]>
-<html class="ie ie7" <?php language_attributes(); ?>>
-<![endif]-->
-<!--[if IE 8]>
-<html class="ie ie8" <?php language_attributes(); ?>>
-<![endif]-->
-<!--[if !(IE 7) & !(IE 8)]><!-->
 <html <?php language_attributes(); ?>>
-<!--<![endif]-->
 <head>
 	<meta charset="<?php bloginfo( 'charset' ); ?>">
 	<meta name="viewport" content="width=device-width">
 	<title><?php wp_title( '|', true, 'right' ); ?></title>
 	<link rel="profile" href="https://gmpg.org/xfn/11">
 	<link rel="pingback" href="<?php echo esc_url( get_bloginfo( 'pingback_url' ) ); ?>">
-	<!--[if lt IE 9]>
-	<script src="<?php echo esc_url( get_template_directory_uri() ); ?>/js/html5.js?ver=3.7.0"></script>
```

Comment:
Remove `js/html5.js` file

Reply:
Please do not delete files in case someone refers to them in a child theme.

Reply:
Oh, I missed that. But if the parent theme does not support IE, then the child theme also needs to be updated, right?

Reply:
I was going to ask about the file deletion, as in the other theme's pull request I was told to keep the files. The way I understand it - if you want to keep the support in your theme, you can, but that's up to you. In any case, I can see why we need to leave the files.

Reply:
is there anything else actionable here? Happy to apply adjustments, if needed.